lib/plugin.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('mz/fs');
+const JSZip = require('jszip');
+const path = require('path');
+const semver = require('semver');
+const Changelog = require('./changelog');
+
+function readInfosFromComposerJson(composerJsonString) {
+    const composerJson = JSON.parse(composerJsonString);
+
+    return {
+        version: composerJson.version,
+        shopwareCompatibility: composerJson.require['shopware/core'],
+        label: {
+            en: composerJson.extra.label['en-GB'],
+            de: composerJson.extra.label['de-DE'],
+        },
+    };
+}
+
+function readInfosFromPluginJson(pluginJsonString) {
```

Comment:
Please see my comment above.
```suggestion
function readInfoFromPluginJson(pluginJsonString) {
```